en/_posts/2016-04-22-marathon-load-balancer.md

```diff
@@ -6,18 +6,73 @@ author: devatoria
 lang: en
 ---
 
-The Marathon load balancer can be configured to use a specific domain name, to make you application URL more user-friendly. For example, let's say we have a nginx application and we want it to be available at `https://myapp.info`.
+In this tutorial, we'll see how to configure TODO [HAProxy](https://cbonte.github.io/haproxy-dconv/intro-1.6.html) load balancer through application labels.
```

Comment:
Oops! Well spotted. I struggled to find the appropriate wording, and forgot about it.